lib/configurators/databases/aws.go

```diff
@@ -353,7 +352,7 @@ func policiesTarget(flags BootstrapFlags, accountID string, identity awslib.Iden
 	if flags.AttachToUser != "" {
 		userArn := flags.AttachToUser
 		if !arn.IsARN(flags.AttachToUser) {
-			userArn = fmt.Sprintf("arn:aws:iam::%s:user/%s", accountID, flags.AttachToUser)
+			userArn = fmt.Sprintf("arn:%s:iam::%s:user/%s", identity.GetPartition(), accountID, flags.AttachToUser)
```

Comment:
I believe this line is breaking the unit test (nil pointer). Not really your fault as the code wasn't designed for a different partition. 

Suggested fix:
`policiesTarget` is called by `buildActions` above. You can do `partitionID` the same way as `accountID` such that when `config.Identity` is nil, use a `*`.